docs/developer/taxes.mdx

```diff
@@ -0,0 +1,356 @@
+---
+title: Taxes
+---
+
+Saleor offers two ways of calculating taxes: Flat Rates and Dynamic Taxes.
+
+Flat Rates allow you to configure static tax rates per country and channel, and assign them to products via tax classes. Dynamic Taxes, on the other hand, enable you to delegate tax calculation to external apps, including 3rd-party tax providers like Avalara, TaxJar, or entirely custom tax apps.
+
+You can configure the tax calculation method per channel and override the configuration for different countries.
+
+## Tax Configuration
+
+The tax configuration object defines various configuration options for a channel. This includes the tax calculation method, whether to charge taxes in this channel, or whether prices are entered including tax. The tax configuration object is created automatically for each sales channel. In the API, it is represented by the `TaxConfiguration` object (see the [API reference](api-reference/objects/tax-configuration.mdx)).
+
+The tax configuration object has several properties, including:
+
+- `chargeTaxes` - Determines whether taxes are charged in the channel. If enabled, taxes are calculated and customers pay the gross prices. If disabled, taxes are still calculated and available for display, but customers pay the net prices.
+- `taxCalculationStrategy` - The strategy to use for tax calculation in the channel. There are two strategies available:
+  - flat rates - the default strategy
+  - tax apps (dynamic taxes)
```

Comment:
```suggestion
  - dynamic taxes (tax apps)
```

I would swap them because this is the naming we use in the first sentence of the document.